src/app/pages/ConsensusAccountDetailsPage/ConsensusAccountDetailsView.tsx

```diff
@@ -0,0 +1,35 @@
+import { FC } from 'react'
+import { Account } from '../../../oasis-nexus/api'
+import { useTranslation } from 'react-i18next'
+import { CardEmptyState } from '../../components/CardEmptyState'
+import { ConsensusAccountData } from '../../components/Account'
+
+type ConsensusAccountDetailsViewProps = {
+  account: Account | undefined
+  isError?: boolean | undefined
+  isLoading?: boolean | undefined
+  showLayer?: boolean
+  standalone?: boolean
+}
```

Comment:
Can we extend this interface from `ConsensusAccountDataProps`, looks to me pretty similar.

Reply:
Done